README.md

```diff
@@ -1 +1,68 @@
-# html5-coding-standards
\ No newline at end of file
+HTML5 Coding Standards
+=====================
+
+HTML5 coding standards you must consider within your projects.
+
+## Table of contents
+
+- [General recommendations](#General-recommendations)
+- [Links and Hyperlinks](#Links-and-Hyperlinks)
+
+## General recommendations
+
+
+### Git Branching Naming
+
+Each branch has its default name, we must set the prefix name as the default name in order to recognize which type of branch are we working in. Example:
+
+**Correct Naming** :ok_hand:
+
+``` feature/cap1-mime-type  ``` 
+
+**Incorrect Naming** :-1:
+
+``` cap1-mime-type  ``` 
+
+### Kebab-case names
+
+When a web browser accesses a root folder, they'll only read camel cases or kebab cases folders to link a path. Otherwise, an error will occur. Example:
+
+**Correct Naming** :ok_hand:
+
+``` feature/cap1-mime-type  ``` 
+
+**Incorrect Naming** :-1:
+
+``` cap1_mime_type  ``` 
+
+## Links and Hyperlinks
```

Comment:
La parte de recomendaciones generales puede quedar en este archivo pero para los específicos de HTML5 hay que generar una Tabla de Contenido con hiperlinks a sus propios README.

Por ejemplo: https://github.com/xfiveco/html-coding-standards#table-of-contents

La sección para Links and Hyperlinks se podría llamar "Attaching CSS and JS files"

Reply:
Ups no le haga caso al cambio que hice,  solo quería ver como se veía se me olvidó que cuando doy commit lo actualiza en automátco. Le iba a preguntar, entonces  el archivo README dentro de el, en la tabla de contenido hago un link que me lleve al archivo de la sección Attaching CSS and JS files?

Reply:
Así es, preferiría que incluso el archivo de la sección Attaching CSS puede quedar como otr README dentro de un folder. Por ejemplo:

```
README.md
\attaching-css-and-js-files/
 \README.md

```
 
Si haces commit en git tus cambios se quedan solo en local, es solo hasta que haces un Push que ya los manda al repositorio remoto.

Reply:
Listo, solo deje revisar los links estos que no se como se hace.